tests/schemes/test_thrift.py

```diff
@@ -1162,3 +1163,80 @@ def testException(request):
     )
 
     assert 1 == res.status_code
+
+
+##############################################################################
+# Calling registered functions directly should be equivalent to calling them
+# as if they weren't registered at all
+
+
+def body(**kwargs):
```

Comment:
Yes, once we settle on an API to let users construct the args structs for different service methods.

I was thinking something like, but let's discuss that elsewhere.

```python
@tchannel.thrift.register
def doFoo(request):
    # ...

def test_do_foo():
    # ...
    doFoo(Request(body=doFoo.args(1, 2, 3)))
```